package.json

```diff
@@ -23,17 +23,9 @@
     "publish:latest": "lerna publish from-git --no-git-reset --no-git-tag-version --no-push",
     "publish:next": "lerna publish preminor --exact --canary --preid next --dist-tag next --no-git-reset --no-git-tag-version --no-push --ignore-scripts --yes",
     "publish:prepare": "lerna version --ignore-scripts --yes --no-push",
-    "start": "yarn browser start",
-    "start:debug": "yarn browser start:debug",
-    "start:integrated": "yarn browser start:integrated",
-    "start:ws": "yarn browser start:ws",
-    "start:ws:debug": "yarn browser start:ws:debug",
     "upgrade:next": "glsp updateNext",
     "watch": "concurrently --kill-others -n tsc,browser -c red,yellow \"tsc -b -w --preserveWatchOutput\" \"yarn -s browser watch\""
```

Comment:
nit: The `watch` only goes to the browser, should that also work for electron? If so we should either split it into `watch:browser` and `watch:electron` or simply add a `watch:electron`.

Reply:
Its now splitted